cigaradvisor/blocks/hero/hero.css

```diff
@@ -15,7 +14,7 @@ main .hero.block {
 }
 
 main .hero.block.under-nav {
-  margin-top: -43px;
+  padding-top: 60px;
```

Comment:
You can't change the padding of the hero like this. It impacts other places.

Reply:
Currently this hero is used in home page and pages like 404, I have verified this change works both these places.